pkg/multiboot/trampoline.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package multiboot
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+)
+
+const (
+	TrampolineEBX = "u-root-ebx-long"
+	TrampolineEP  = "u-root-ep-quad"
+)
+
+func (m *Multiboot) addTrampoline() error {
+	// Trampoline setups the machine registers to desired state
+	// and executes the loaded kernel.
+	d, err := setupTrampoline(m.trampoline, m.InfoAddr, m.KernelEntry)
```

Comment:
can't leave this comment elsewhere -- can you call the trampoline file trampoline-amd64? or something with the arch name in it.